presentation/src/main/java/com/exallium/mvvmcleanexample/presentation/utils/notifiable/BaseNotifiableObservable.kt

```diff
@@ -0,0 +1,67 @@
+package com.exallium.mvvmcleanexample.presentation.utils.notifiable
+
+import android.databinding.Bindable
+import android.databinding.Observable
+import android.databinding.PropertyChangeRegistry
+
+class BaseNotifiableObservable : NotifiableObservable {
```

Comment:
I can remove this class, we probably don't need it. This is merely a work around to not enforce having to inherit from the framework version of BaseObservable and still get all the features from it. This is a re-implementation of it that allows us to delegate the behaviour to it. That or you implement Observable yourself and reimplement the PropertyChangeRegistry boilerplate every time.